packages/worker/src/npmRegistry/api.ts

```diff
@@ -55,14 +68,33 @@ export async function fetchPackageMetadata(name: string) {
   // );
 
   return {
-    description: document.description ?? undefined,
-    license: document.license ?? undefined,
+    description: document.description,
+    fullDescription: document.readme,
+    maintainers: (document.maintainers ?? []).map((author) => ({
+      name: author.name,
+      email: author.email,
+      avatar: `https://s.gravatar.com/avatar/${createHash('md5')
+        .update(author.email)
+        .digest('hex')}`,
+    })),
+    tags: document.keywords ?? [],
+    license: document.license,
     homepageUrl,
     repositoryUrl,
     updateSeq: Number(document._local_seq),
     updatedAt: new Date(document.time.modified),
     latestVersion: document['dist-tags'].latest,
     versionList,
+    versionSpecificValues: versionList.reduce((acc, el) => {
```

Comment:
nit: what if we make a single `versions: Record<string, VersionData>` property? So the `versionList` can be easily obtained by `Object.keys(versions)`

Reply:
Or `versionMap`